dichotomy-api/src/main/java/com/farao_community/farao/dichotomy/api/ValidationException.java

```diff
@@ -0,0 +1,11 @@
+package com.farao_community.farao.dichotomy.api;
+
+/**
+ * @author Joris Mancini {@literal <joris.mancini at rte-france.com>}
+ */
+public class ValidationException extends Exception {
```

Comment:
Using this exception will hide checked exceptions  that should be handled directly  by developer (example automatically closing IO ressource).
extend RuntimeException to wrap only unchecked exception.